README.md

```diff
@@ -43,324 +43,6 @@ The documentation should help configure the BinderHub service to:
 [persistent binderhub chart]: https://github.com/gesiscss/persistent_binderhub
 [was added]: https://github.com/jupyterhub/binderhub/pull/666
 
-## Installation
```

Comment:
I would maybe still leave the "Installation" heading here, but have a link to point to the rendered readthedocs site. Only because I think there is an expectation of many projects that installation instructions are available in the README, so providing a link will get people to where they need to go :)